web/src/popover_menus.js

```diff
@@ -272,6 +277,92 @@ export function initialize() {
         },
     });
 
+    tippy_no_propagation("#stream_filters .topic-sidebar-menu-icon", {
+        ...left_sidebar_tippy_options,
+        onShow(instance) {
+            popover_instances.topics_menu = instance;
+            on_show_prep(instance);
+            const elt = $(instance.reference).closest(".topic-sidebar-menu-icon").expectOne()[0];
+            instance.context = popover_menus_data.get_topic_popover_content_context(elt);
+            instance.setContent(parse_html(render_topic_sidebar_actions(instance.context)));
+        },
+        onMount(instance) {
+            const $popper = $(instance.popper);
+            const {stream_id, topic_name} = instance.context;
+
+            if (!stream_id) {
+                instance.hide();
+                return;
+            }
+
+            $popper.one("click", ".sidebar-popover-mute-topic", (e) => {
+                muted_topics_ui.mute_topic(stream_id, topic_name);
+                e.stopPropagation();
```

Comment:
Doesn't the `tippy_no_propagation` module do this automatically?

Reply:
I have removed it. However, most of the click handlers inside popover_menus include the calls `e.stopPropagation` and `e.preventDefault`. Since `tippy_no_propagation` already handles these cases, I will raise another PR to refactor popover_menus by carefully evaluating and removing this unnecessary piece of code.